usr/src/uts/intel/os/minor_perm

```diff
@@ -269,3 +269,44 @@ timerfd:* 0666 root sys
 signalfd:* 0666 root sys
 iwn:* 0666 root sys
 clone:iwn 0666 root sys
+arn:* 0666 root sys
```

Comment:
Check the top-level directory's `manifest` file to see if we ship this driver at all.  For THIS one we don't, e.g.  There may be others in this list that don't.

Reply:
That indeed makes sense (header files and fwflash libraries, though maybe we should at some point examine THOSE as well?).

Reply:
So should `hermon` (and anything else not actually shipping a driver) be removed from this change as well?

Reply:
Yes, actually, you should not have a hermon minor_perm entry here.  Good catch, and my above comment wasn't supposed to go out until the rest of these did.

Reply:
Only drivers that have a corresponding entry in `manifest` remain.